server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -1217,6 +1218,30 @@ public void assertMatched() {
         }
     }
 
+    public void testReconfiguresToExcludeMasterIneligibleVotingNodes() {
```

Comment:
`testReconfiguresToExcludeMasterIneligibleNodesInVotingConfig`

Reply:
As you wish :) see eca9468a03fafc8a367ad9937653d4b81c833dd8